packages/integration-tests/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -239,7 +239,13 @@ export function runtimeIntegrationTest<Client>(input: Input<Client>) {
       const { ctx, state } = await setupScenario(kind, input, scenario)
       states[scenario.name] = state
 
-      const PrismaClient = await getTestClient(ctx.fs.cwd())
+      await generateInFolder({
```

Comment:
Changed this `getTestClient` call to `generateInFolder`. This has currently 2 problems:

- The test fails with OOM error (if I reduces the number of scenarios, then they pass.)
- The assertions about Decimal don't pass because the Decimal constructors don't match

Reply:
I see, let's keep your original work, I hoped this would not get too tricky. Thanks for trying!

Reply:
(these tests are also legacy tests that we will eventually get rid of)

Reply:
Ok, I'll move this back to `sql-template-tag@4.1.0 + type modification` version